<!-- Tests Pepper's motion control -->
<launch>

    <!-- <arg name="pepper_ip" default="169.254.5.105"/> -->
    <arg name="pepper_ip" default="138.67.202.228"/>
    <arg name="pepper_port" default="9559"/>
    <arg name="roscore_ip" default="169.254.13.225"/>
    <arg name="network_interface" default="eno1"/>

    <!--
    <include file="$(find pepper_dcm_bringup)/launch/pepper_dcm_bringup_trajectory.launch">
        <arg name="robot_ip" value="$(arg pepper_ip)"/>
        <arg name="roscore_ip" value="$(arg roscore_ip)"/>
        <arg name="network_interface" value="$(arg network_interface)"/>
    </include>


    <include file="$(find pepper_bringup)/launch/pepper_full_py.launch">
        <arg name="nao_ip" value="$(arg pepper_ip)"/>
    </include>
    -->

    <!--
    <include file="$(find pepper_bringup)/launch/pepper_full.launch">
        <arg name="nao_ip" value="$(arg pepper_ip)"/>
        <arg name="roscore_ip" value="$(arg roscore_ip)"/>
        <arg name="network_interface" value="$(arg network_interface)"/>
    </include>
    -->

    <node name="test_control" pkg="pepper_vr_teleop" type="test_control.py" output="screen">
        <param name="control_type" value="almotion"/>
        <param name="robot_ip" value="$(arg pepper_ip)"/>
        <param name="port" value="$(arg pepper_port)"/>
    </node>

</launch>
